deployment/argo-workflows/eks/workflow-planet-scene-id.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: workflow-
+spec:
+  entrypoint: workflow-template
+  parallelism: 30
+  arguments:
+    parameters:
+    - name: activator-input
+      value: |
+        ["20160831_143848_0c79"]
+
+  templates:
+  - name: workflow-template
+    dag:
+      tasks:
+      - name: activator
+        template: activator-template
+        arguments:
+          parameters:
+          - name: activator-input
+            value: "{{item}}"
+        withParam: "{{workflow.parameters.activator-input}}"
+
+  - name: activator-template
+    inputs:
+      parameters:
+      - name: activator-input
+    # https://argoproj.github.io/argo-workflows/fields/#backoff
+    retryStrategy:
+      limit: "10"
+      backoff:
+        duration: "10m" # Default unit is seconds. Could also be a Duration, e.g.: "2m", "6h", "1d"
+        factor: "2"
+        # maxDuration: "1m" Default unit is seconds. Could also be a Duration, e.g.: "2m", "6h", "1d"
```

Comment:
@jamesmcclain do you think setting it to "1d" is a reasonable expectation? Or you think it will be convenient to await for a longer time and not to be limited by maxDuration (this is the max duration with all retries)

Reply:
Yes, I think that one day is a reasonable period between the first and second attempt (is there a way to predicate the behavior on the specific failure?).  Whatever is typical for other imagery between the nth and (n+1)st attempt is probably also reasonable for this imagery type.